python-backend/app/api/mines/namespace/mines.py

```diff
@@ -18,6 +19,7 @@
 api.add_resource(MineTypeDetailResource, '/mine_types/details', '/mine_types/details/<string:mine_type_detail_xref_guid>')
 api.add_resource(MineTenureTypeCodeResource, '/mine_tenure_type_codes')
 api.add_resource(MineDisturbanceCodeResource, '/disturbance_codes')
+api.add_resource(MineCommodityCodeResource, '/commodity_codes')
 api.add_resource(MineLocationResource, '/location', '/location/<string:mine_location_guid>')
 api.add_resource(MineStatusResource, '/status', '/status/<string:mine_status_guid>')
 api.add_resource(MineRegionResource,'/region','/region/<string:mine_region_guid>')
```

Comment:
We should be using kebab case for url paths as that is the http standard.

/commodity-codes
/disturbance-codes
/mine-tenure-type-codes

Reply:
Sure